collated/main/A0124333U.md

```diff
@@ -0,0 +1,822 @@
+        return this.newConfig;
+    }
+
+    @Override
+    public String toString() {
+        return "File Path changed to " + this.newFilePath;
+    }
+
+}
+```
+###### \java\tars\commons\events\ui\KeyCombinationPressedEvent.java
+``` java
+ */
+public class KeyCombinationPressedEvent extends BaseEvent {
+    
+    private KeyCombination keyComb;
+    
+    public KeyCombinationPressedEvent(KeyCombination keyComb) {
+        this.keyComb = keyComb;
+    }
+    
+    public KeyCombination getKeyCombination() {
+        return keyComb;
+    }
+
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName();
+    }
+
+}
+```
+###### \java\tars\commons\util\DateTimeUtil.java
+``` java
+     * @param dateTimeSource
+     * @param dateTimeQuery
+     */
+    public static boolean isDateTimeWithinRange(DateTime dateTimeSource, DateTime dateTimeQuery) {
+        boolean isTaskDateWithinRange = true;
+
+        // Return false if task is a floating task (i.e. no start or end
+        // dateTime
+        if (dateTimeSource.getEndDate() == null) {
+            return false;
+        }
+
+        // Case 1: dateTimeQuery has a range of date (i.e. startDateTime &
+        // endDateTime != null)
+        if (dateTimeQuery.getStartDate() != null) {
+
+            if (dateTimeSource.getEndDate().isBefore(dateTimeQuery.getStartDate())) {
+                return false;
+            }
+            
+            // Case 1a: dateTimeSource has a range of date 
+            if (dateTimeSource.getStartDate() != null) {
+                if (dateTimeSource.getStartDate().isAfter(dateTimeQuery.getEndDate())) {
+                    return false;
+                }
+            } else {  //Case 1b: dateTimeSource only has a endDateTime
+                if (dateTimeSource.getEndDate().isAfter(dateTimeQuery.getEndDate())) {
+                    return false;
+                }
+            }
+        } else { // Case 2: dateTimeQuery only has a endDateTime
+
+            // Case 2a: dateTimeSource has a range of date  
+            if (dateTimeSource.getStartDate() != null) {
+                if (dateTimeQuery.getEndDate().isBefore(dateTimeSource.getStartDate())
+                        || dateTimeQuery.getEndDate().isAfter(dateTimeSource.getEndDate())) {
+                    return false;
+                }
+            } else { //Case 2b: dateTimeSource only has a endDateTime
+                if (!dateTimeQuery.getEndDate().equals(dateTimeSource.getEndDate())) {
+                    return false;
+                }
+            }
+        }
+
+        return isTaskDateWithinRange;
+    }
+
+    /**
+     * Modify the date based on the new hour, min and sec
+     * 
+```
+###### \java\tars\logic\commands\CdCommand.java
+``` java
+ */
+
+public class CdCommand extends Command {
+
+    public static final String COMMAND_WORD = "cd";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Changes the directory of the "
```

Comment:
Avoid lines longer than 80 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.